tests/testthat/test-metadata_to_coldata.R

```diff
@@ -57,7 +57,8 @@ sce_list <- purrr::imap(
 # create merged sce
 merged_sce <- merge_sce_list(
   sce_list = sce_list,
-  retain_coldata_cols = "sum"
+  retain_coldata_cols = "sum",
+  include_altexp = FALSE
```

Comment:
This should work without modification, right?
```suggestion
  retain_coldata_cols = "sum"
```